packages/election/migrations/sqls/20180912181821-election-up.sql

```diff
@@ -0,0 +1,89 @@
+CREATE EXTENSION IF NOT EXISTS "uuid-ossp";
+
+create table if not exists "elections" (
+  "id"           uuid primary key not null default uuid_generate_v4(),
+  "slug"         varchar          not null,
+  "description"  varchar          not null,
+  "beginDate"    timestamptz      not null,
+  "endDate"      timestamptz      not null,
+  "numSeats"     integer          not null,
+  "discussionId" uuid             not null references "discussions" on update cascade on delete cascade,
+  "active"       boolean          not null default true,
+  "result"       jsonb,
+  "createdAt"    timestamptz               default now(),
+  "updatedAt"    timestamptz               default now(),
+  unique ("slug")
+);
+
+create table if not exists "electionCandidacies" (
+  "id"             uuid primary key not null default uuid_generate_v4(),
+  "userId"         uuid             not null references "users"  on update cascade on delete cascade,
+  "electionId"     uuid             not null references "elections" on update cascade on delete cascade,
+  "commentId"      uuid             references "comments",
+  "recommendation" varchar,
+  "createdAt"      timestamptz               default now(),
+  "updatedAt"      timestamptz               default now(),
+  unique ("userId", "electionId")
+);
+
+create table if not exists "electionBallots" (
+  "id"          uuid primary key not null default uuid_generate_v4(),
+  "userId"      uuid             not null references "users" on update cascade on delete cascade,
+  "candidacyId" uuid             not null references "electionCandidacies" on update cascade on delete cascade,
+  "createdAt"   timestamptz               default now(),
+  "updatedAt"   timestamptz               default now(),
+  unique ("userId", "candidacyId")
+);
```

Comment:
this table should not have been created, as the API for it is not present in this PR. It turned out the table needed changes in order to be able to serve the API.